internal/log/option.go

```diff
@@ -0,0 +1,20 @@
+package log
+
+type Option func(*option)
+
+type option struct {
+	logger Logger
+}
+
+func (o *option) evaluateOption(opts ...Option) *option {
```

Comment:
please write correct code, we don't need evaluateOption

Reply:
sorry:bow:
I fixed to `apply`:bow: